kalite/testing/base_environment.py

```diff
@@ -245,3 +254,42 @@ def do_fake_registration():
     device = Device.get_own_device()
     device_zone = DeviceZone(device=device, zone=zone)
     device_zone.save()
+
+
+def _make_video(context):
+    lang_code = "en"
+    youtube_id = "my_cool_id"
+    item_dict = {
+        "title": "Subtitled Video",
+        "description": "A video with subtitles",
+        "available": True,
+        "kind": "Video",
+        "id": "video_with_subtitles",
+        "slug": "video_with_subtitles",
+        "path": "khan/video_with_subtitles",
```

Comment:
Yep, `make assets` gets run before tests (https://github.com/learningequality/ka-lite/blob/0.16.x/circle.yml#L21), meaning that this line (https://github.com/learningequality/ka-lite/blob/0.16.x/Makefile#L100) is executed, and a minimal language pack is available.